### PR TITLE
Adding nerdbabe.yml file :)

### PR DESCRIPTION
**Pull request by nerdbabe**

### CODE REVIEW

**Comment by moonglum** [comment]
Niiiice :+1: Welcome :bowtie:

File: config/travellers/nerdbabe.yml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    - "Fredericia"
+    - "Fehmarn"
+    - "Ghent"
+    - "Glasgow"
+    - "Gelsenkirchen"
+    - "Hamburg"
+    - "Hanover"
+    - "Heidelberg"
+    - "Helgoland"
+    - "Inverness"
```
Comment (skade) [inline config/travellers/nerdbabe.yml]: We should talk about Scotland sometime...

**Comment by bascht** [comment]
Welcome aboard, fellow traveller! :anchor:

File: config/travellers/nerdbabe.yml (hunk 1 above)
Comment (moonglum) [inline config/travellers/nerdbabe.yml]: Oh scotland :heart:

Comment (nerdbabe) [inline config/travellers/nerdbabe.yml]: been there 2 times. One time with my bike.

Comment (skade) [inline config/travellers/nerdbabe.yml]: I was hiking there for three weeks. West Highland way, parts of the Great Glen way and then up to the Skye.

Comment (nerdbabe) [inline config/travellers/nerdbabe.yml]: maybe we should do a ruby hiking thing in scotland? :)

Comment (moonglum) [inline config/travellers/nerdbabe.yml]: Will go hiking with @13abylon through Scotland this year :+1: Yay for Ruby hiking :wink:

**Pull request merged**